mesa/Person.py

```diff
@@ -215,8 +215,8 @@ def init_person(self): # person command
         # notice how this deforms a little the initial setup
         self.education_level = self.max_education_level
         for level in sorted(list(self.model.education_levels.keys()), reverse=True):
-            max_age = self.model.education_levels.get(level)[1]
-            if self.age() <= max_age:
+            max_age = self.model.education_levels[level][1]
+            if self.age() <= max_age or self.education_level > self.max_education_level:
```

Comment:
This has already been solved in the PR #202, issue #201, this condition was added because some agents earned a education-level higher than max-education-level.